EventFilter/CastorRawToDigi/plugins/CastorRawToDigi.cc

```diff
@@ -21,7 +21,7 @@ using namespace std;
 CastorRawToDigi::CastorRawToDigi(edm::ParameterSet const& conf):
   dataTag_(conf.getParameter<edm::InputTag>("InputLabel")),
   unpacker_(conf.getParameter<int>("CastorFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
-  zdcunpacker_(conf.getParameter<int>("ZDCFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
+  zdcunpacker_(conf.getParameter<int>("CastorFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
```

Comment:
is it really used?
I don't see it anywhere in https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_7/EventFilter/CastorRawToDigi/src/ZdcUnpacker.cc